notebooks/tutorials/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,176 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: Using the stac_ipyleaflet mapping library\n",
+    "description: This project leverages the [ipyleaflet](https://pypi.org/project/ipyleaflet/) mapping library and is intended to reduce the code needed to visualize data from the [VEDA STAC](https://staging-stac.delta-backend.com/collections). The beta version supports Cloud-Optimized GeoTIFFs (COGs).\n",
+    "  Currently, this package works within [this JupyterHub Environment](https://nasa-veda.2i2c.cloud/).\n",
+    "author: Emma Paz\n",
+    "date: September 11, 2023\n",
+    "execute:\n",
+    "  freeze: true  \n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualizing STAC Data using stac_ipyleaflet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run This Notebook\n",
+    "It is recommended to run through this tutorial within VEDA's **Pangeo Notebook**, which already includes the stac_ipyleaflet package.\n",
```

Comment:
Might be good to add a quick section on how to select the image through the notebook kernel selector. They'll need `nasa-veda-singeruser` kernel
<img width="470" alt="Screen Shot 2023-09-11 at 8 50 16 AM" src="https://github.com/NASA-IMPACT/veda-docs/assets/208859/77c19aa8-a200-4d39-9b89-ecb261cf9c8c">